### PR TITLE
Do not allow negative `fluxerr` values in photometry POST handler

### PR DESCRIPTION
**Pull request by acrellin**
Closes https://github.com/skyportal/skyportal/issues/1861
Closes https://github.com/skyportal/skyportal/issues/1839

### CODE REVIEW

**Review by kmshin1397 — APPROVED** [review]

File: skyportal/handlers/api/photometry.py [hunk 1]
```diff
@@ -173,6 +173,14 @@ def standardize_photometry_data(self):
                 )
             else:
                 kind = 'flux'
+                if (
```
Comment (stefanv) [inline skyportal/handlers/api/photometry.py:176]: The documentation of `PhotFluxFlexible` says

>     """This is one of two classes used for rendering the
    input data to `PhotometryHandler.post` in redoc. These classes are only
    used for generating documentation and not for validation, serialization,
    or deserialization."""

I guess that's not true!

Comment (stefanv) [reply skyportal/handlers/api/photometry.py:176]: Should these checks be specified in `schema.py` using the [validate keyword](https://marshmallow.readthedocs.io/en/stable/marshmallow.fields.html#marshmallow.fields.Method)?

**Review by dmitryduev — APPROVED** [review]
LGTM!

File: skyportal/handlers/api/photometry.py (hunk 1 above)
Comment (acrellin) [reply skyportal/handlers/api/photometry.py]: Agreed -- updated

**Review by stefanv — APPROVED** [review]
Nicely done!

Should the validation exception now be caught in the handler?

**Comment by acrellin** [comment]
@stefanv yes, it is already manually caught

**Pull request merged**